launchUI.py

```diff
@@ -1311,7 +1323,7 @@ def drawSchedule():
 
     for ii in range(mappingII):
         # draw data memory
-        spmLabel = tkinter.Label(canvas, text="Data\nSPM", fg='black', bg='gray', relief='raised', bd=BORDER)
+        spmLabel = tkinter.Label(canvas, text="Data\nSPM", fg='black', bg='gray', relief='raised', bd=BORDER, highlightbackground="black", highlightthickness=2)

```

Comment:
Would it look better if we make the border a little thinner? Say `thickness = 1`? WDYT?

Reply:
And I think the `thickness` could also be a global constant variable, right?

Reply:
Certainly, I set the thickness as a global variable and I set the thickness to 1. It seems that let thickness = 1 will make the graphical user interface more clear and concise.
![Uploading GUI thickness=1.png…]()